src/tv2-common/helpers/graphics/caspar/__tests__/htmlPilotGraphicGenerator.spec.ts

```diff
@@ -0,0 +1,193 @@
+import { IBlueprintRundownDB, PieceLifespan, PlaylistTimingType, TSR } from 'blueprints-integration'
+import { CueDefinitionGraphic, GraphicPilot, HtmlPilotGraphicGenerator, literal } from 'tv2-common'
+import { CueType, SharedGraphicLLayer } from 'tv2-constants'
+import { SegmentUserContext } from '../../../../../__mocks__/context'
+import { defaultShowStyleConfig, defaultStudioConfig } from '../../../../../tv2_afvd_showstyle/__tests__/configs'
+import { getConfig, parseConfig as parseShowStyleConfig } from '../../../../../tv2_afvd_showstyle/helpers/config'
+import { parseConfig as parseStudioConfig } from '../../../../../tv2_afvd_studio/helpers/config'
+import mappingsDefaults from '../../../../../tv2_afvd_studio/migrations/mappings-defaults'
```

Comment:
note to reviewers: these tests depend on afvd defaults and configs because they're already there, and technically both blueprints are still cross-compatible with the HTML and Viz graphics, but it's a good idea to refactor out this dependency in the future